### DISCUSSION TITLE
GitHub Clone with Beer.css

### DISCUSSION THREAD

**Discussion by w3teal** [beercss/beercss#328] [Show and tell]
Beer.css was the most fun thing i found in styling. For example, i can make an nice-looking GitHub Clone in less than 1 day! This is the result : https://codepen.io/LIGMATV/full/pomMbZE (It not responsive in mobile, because the point is for fun)

**Comment by leonardorafael** [comment]
![8vnzlv](https://github.com/beercss/beercss/assets/12979016/733b69b3-e212-428b-b1bb-c9ca938f5f71)

**Comment by lighth7015** [comment]
Now if I could find time to do that for [Paperbase](https://codesandbox.io/p/sandbox/material-ui-paperbase-t614v), lol.

**Reply by w3teal** [reply]
It seems cool.

**Reply by lighth7015** [reply]
It is, it would be nice to have a Firebase-style UI written around beercss, lol.

**Reply by gobijan** [reply]
Out of curiosity I started it and came to this result in 20 minutes. Not having much time but maybe you wanna go on with it?
![CleanShot 2024-07-31 at 12 46 25](https://github.com/user-attachments/assets/1a43f9ef-ee2c-4b70-a3c2-911c3a822c65)

https://codepen.io/gobijan/pen/ZEdezvj

**Reply by gobijan** [reply]
Found a few minutes and updated the fiddle. Let me know what you think :) 

![CleanShot 2024-08-08 at 16 20 24](https://github.com/user-attachments/assets/08e2174f-462b-44cc-b0ed-7493c3e163e4)

**Reply by lighth7015** [reply]
Thank you! :D

**Comment by leonardorafael** [comment]
![934btt](https://github.com/user-attachments/assets/ba36b11a-88ef-4594-a7ea-d2b4b839ec04)

**Comment by coffeeandwork** [comment]
> Beer.css was the most fun thing i found in styling. For example, i can make an nice-looking GitHub Clone in less than 1 day! This is the result : https://codepen.io/LIGMATV/full/pomMbZE (It not responsive in mobile, because the point is for fun)

This is awesome!
